### PR TITLE
Feature/help

### PR DESCRIPTION
**Pull request by david-kasali**

### CODE REVIEW

File: pages/operator-company-details.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const LimitedCompanyDetails = (props) => (
           }}
           errorPrefix={`${props.t("Error")}: `}
           hint={props.t(
-            "Every registered company will have a Companies House reference number."
+            "Every registered company will have a Companies House reference number which will usually look like this Eight numbers (e.g. 12345678) or two letters followed by 6 numbers (e.g. AB123456)"
```
Comment (paul-burnett) [inline pages/operator-company-details.js:57]: Would it be better to say:
"Every registered company will have a Companies House reference number which will usually be either eight numbers (e.g. 12345678) or two letters followed by six numbers (e.g. AB123456)"

File: pages/establishment-address.js [hunk 2]
```diff
@@ -58,6 +58,7 @@ const EstablishmentAddress = (props) => (
               touched: true,
               error: props.t(props.validatorErrors.establishment_postcode_find)
             }}
+            hint={props.t("For example, SW1H 9EX")}
```
Comment (paul-burnett) [inline pages/establishment-address.js:61]: Any reason why you chose that example postcode?

Comment (david-kasali) [reply pages/establishment-address.js:61]: FSA HQ's postcode

Comment (paul-burnett) [reply pages/establishment-address.js:61]: Great :)

**Review by paul-burnett — APPROVED** [review]

**Comment by paul-burnett** [comment]
Closing PR as changes incorporated into new frontend migration

**Pull request closed without merging**